generators/dart/colour.ts

```diff
@@ -25,7 +25,9 @@ export function colour_picker(block: Block, generator: DartGenerator): [string,
 
 export function colour_random(block: Block, generator: DartGenerator): [string, Order] {
   // Generate a random colour.
-  generator.definitions_['import_dart_math'] =
+  // TODO(#7600): find better approach than casting to any to override
```

Comment:
Similarly to before (I think) would like this linked to #7326 because I think this is an API problem not just a type safety problem.

Reply:
I believe you indicated you were statisfied with #7600 linking to #7326 about this topic (which it does).  Would you like the actual TODO to do so as well?

Reply:
Oh haha nice I had forgotten. yep sg